src/index.ts

```diff
@@ -276,6 +280,28 @@ async function uploadSecrets() {
 	}
 }
 
+function extractDeploymentUrlsFromStdout(stdOut: string): {
```

Comment:
Wait why are parsing these things from stdout? This should be in wrangler artifacts, no?

Reply:
Added a comment // fallback to trying to extract the deployment-url and deployment-alias-url from stdout for wranglerVersion < 3.81.0